requirements/base.txt

```diff
@@ -30,7 +30,7 @@ pyopenssl==16.2.0
 ndg-httpsclient==0.4.0
 pyasn1==0.1.7
 wtforms==2.1
-postcodeinfo>=0.0,<0.1
+postcodeinfo>=0.0,<0.3
```

Comment:
@ministryofjustice/laa-get-access We no longer use `postcodeinfo`, right? Can this dependency be removed?